applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -35,6 +35,14 @@ config BT_AUDIO_CONCURRENT_RX_STREAMS_MAX
 	  The maximum number of concurrent streams supported in the RX direction.
 	  This is the maximum number of streams that can be active at the same time.
 
+config BT_AUDIO_CONCURRENT_TX_STREAMS_MAX
+	int "Max number of concurrent streams"
+	range 1 32
```

Comment:
`range 1 AUDIO_DECODE_CHANNELS_MAX` ?

Reply:
This Kconfig should know nothing about the audio system. We have some use cases where we only need this bluetooth part and omit the audio system all together.